benchmark/websocket/simple.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const common = require('../common.js');
+const crypto = require('crypto');
+const http = require('http');
+const { WebSocket } = require('undici');
+
+const GUID = '258EAFA5-E914-47DA-95CA-C5AB0DC85B11';
+
+const configs = {
+  size: [64, 16 * 1024, 128 * 1024, 1024 * 1024],
+  useBinary: ['true', 'false'],
```

Comment:
```suggestion
  useBinary: [true, false],
```

Reply:
I find that in the 'common.js', the type can only be `number` or `string`.

https://github.com/nodejs/node/blob/3e14cfbbcfcedbb18235ffd5cc28983ac5d5ca52/benchmark/common.js#L131C1-L135C10

Reply:
Ok, we can use 0 and 1, or change `useBinary` to `opcode` (`useBinary: ['true', 'false']` -> `opcode: [1, 2]`). I'm also ok with it as is.